src/components/views/ScreeningsList.vue

```diff
@@ -0,0 +1,54 @@
+<script>
+import { defineComponent } from 'vue';
+import ScreeningsCard from '@/components/views/ScreeningsCard.vue';
+
+export default defineComponent({
+  components: { ScreeningsCard },
+  props: {
+    movies: {
+      type: Array,
+      required: true,
+    },
+    daySelected: {
+      type: Date,
+      default: new Date().toISOString().slice(0, 10)
+    },
+    movieSelected: {
+      type: String,
+      default: 'All movies',
+    }
+  },
+  data() {
+    return {
+      screenings: []
+    }
+  },
+  computed: {
+    filteredMovies() {
+      if (this.movieSelected === 'All movies') {
+        return this.movies
+      } else {
+        return this.movies.filter(movie => this.movieSelected === movie.title)
+      }
```

Comment:
If there is a `return` in a condition, we don't need to wrap the rest of the code in `else { }` 😄 
```suggestion
      return this.movies.filter(movie => this.movieSelected === movie.title)
```